src/diffusion_for_multi_scale_molecular_dynamics/models/score_networks/score_network.py

```diff
@@ -62,11 +72,15 @@ def _check_batch(self, batch: Dict[AnyStr, torch.Tensor]):
         those inputs have the expected dimensions.
 
         It is expected that:
-            - the relative coordinates are present and of shape [batch_size, number of atoms, spatial_dimension]
+            - an AXL namedtuple is present with
+              - the relative coordinates of shape [batch_size, number of atoms, spatial_dimension]
+              - the atom types of shape [batch_size, number of atoms]
+              - the unit cell vectors  TODO shape
             - all the components of relative coordinates will be in [0, 1)
+            - all the components of atom types are integers between [0, number of atomic species)
```

Comment:
This comment is inconsistent with the test on line 161. Maybe change the comment to say:
`number of atomic species` -> `number of atomic classes, including the MASK token` ?